plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy

```diff
@@ -94,25 +89,7 @@ class SamplesheetConverter {
             increaseCount()
 
             Map<String,String> row = fullRow.findAll { it.value != "" }
-            JSONObject jsonRow = new JSONObject(row)
-
-            try {
-                this.validator.performValidation(schema, jsonRow)
-            } 
-            catch (ValidationException e) {
-                if(e.getCausingExceptions().size() > 0){
-                    e.getCausingExceptions().each { this.errors << addSample("${it.getMessage()}".toString()) }
-                }
-                else {
-                    this.errors << addSample("${e.getMessage()}".toString())
-                }
-            }
-            catch (SchemaException e) {
-                this.schemaErrors << e.getMessage()
-            }
-
```

Comment:
I don't get how the samplesheet values are validated if you removed this here? Or am I mising something?

Reply:
I removed this because it is already validated with `validateParameters`. Or were you thinking of performing the validation twice? 
In case someone doesn't want to use `validateParameters` it is true that the values won't be validated, but this shouldn't be encouraged I think.

Reply:
Aaah I get it! So you use the schema specified in the parameters schema to already validate the samplesheet. I think it's a bit weird though that we validate the samplesheet in two parts now, but yeah if both functions are used it shouldn't be a problem :)

Reply:
`unique` and `dependentRequired` are purely for validation and are not used for channel construction (only `type`, `format` and `meta` are used for the construction of a channel). But yeah it doesn't matter that much. Maybe should add a warning to the README that `validateParameters()` always has to be run before `validateAndConvertSamplesheet()`.